src/main/java/org/reactome/qa/doisuggester/Main.java

```diff
@@ -50,213 +56,201 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 			Map<GKInstance, Set<Map<GKInstance, GKInstance>>> pathway2Reactions2NewIEs = new HashMap<>();
 
 			// Iterate through all ReactionlikeEvents (RlEs) in the 'current' test_reactome
-			for (GKInstance currentRLE : (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent))
+			Collection<GKInstance> RLEs = (Collection<GKInstance>) dbAdaptor.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent);
+			System.out.println(RLEs.size() + " Reaction-like events will be checked.");
+			for (GKInstance currentRLE : RLEs)
 			{
 
 				// Only check RlEs that are not found in the 'inferredFrom' of any instances.
 				if (currentRLE.getReferers(ReactomeJavaConstants.inferredFrom) == null)
 				{
 
 					GKInstance previousRLE = dbAdaptorPrev.fetchInstance(currentRLE.getDBID());
+					// If no previous version of the RLE, add it to the New RLE list (to be processed later). 
 					if (previousRLE == null)
 					{
-
+						// An RLE is "new" if there is no equivalent RLE in the previous database AND if it is referred to by a "hasEvent" attribute of another object. 
 						if (currentRLE.getReferers(ReactomeJavaConstants.hasEvent) != null)
 						{
 							newRLEs.add(currentRLE);
 						}
 						// New RLEs have their own logic below, so skip the current loop that deals with
 						// existing RlEs.
-						continue;
+//						continue;
```

Comment:
This continue doesn't seem to be needed since nothing else happens in the loop after it.  Can it be removed?

Reply:
Yes, it will be removed eventually. At the moment, it's a visual marker for me.